lxd/device/gpu_physical.go

```diff
@@ -104,7 +104,7 @@ func (d *gpuPhysical) Start() (*deviceConfig.RunConfig, error) {
 // startCDIDevices starts all the devices given in a CDI specification:
 // * `unix-char` (representing the card and non-card devices)
 // * `disk` (representing the mounts)).
-func (d *gpuPhysical) startCDIDevices(configDevices *cdi.ConfigDevices, hooks *cdi.Hooks, runConf *deviceConfig.RunConfig) error {
+func (d *gpuPhysical) startCDIDevices(configDevices *cdi.ConfigDevices, runConf *deviceConfig.RunConfig) error {
```

Comment:
we shouldnt see several commits that alter the same new function in a single PR, please make your commits final logical commits rather than an evolution during the cycle of the PR.

Reply:
fixed